vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/util/proxy/dial_test.go

```diff
@@ -96,7 +96,7 @@ func TestDialURL(t *testing.T) {
 			ts.TLS = &tls.Config{Certificates: []tls.Certificate{cert}}
 			ts.StartTLS()
 
-			tlsConfigCopy := utilnet.CloneTLSConfig(tc.TLSConfig)
+			tlsConfigCopy := tc.TLSConfig
```

Comment:
@liggitt thanks, what is the proper way to backport that? cherry-pick and replace the Clone calls with utilnet.CloneTLSConfig? Or manually change with a drop commit?